script.js

```diff
@@ -1,4 +1,127 @@
+// Global var for gamemode
+var rollDice = "rollDice";
+var chooseDice = "chooseDice";
```

Comment:
There is something know as [SCREAM_CASE constants ](https://www.freecodecamp.org/news/when-to-capitalize-your-javascript-constants-4fabc0a4a4c4/)

I can touch more on this as the days go by, but it's simply another way to name variables that remain unchanged through out the program eg.
```
var ROLL_DICE_MODE = "rollDice";
```